src/Sylius/Bundle/ApiBundle/StateProcessor/Command/CommandAwareInputDataProcessor.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\StateProcessor\Command;
+
+use AllowDynamicProperties;
+use ApiPlatform\Metadata\Operation;
+
+#[AllowDynamicProperties]
+/* @experimental */
+final class CommandAwareInputDataProcessor implements CommandAwareInputDataProcessorInterface
+{
+    /** @param CommandDataProcessorInterface[] $commandDataProcessors */
+    public function __construct(
+        iterable $commandDataProcessors,
+    ) {
+        $this->commandDataProcessors = $commandDataProcessors;
+    }
```

Comment:
```suggestion
/* @experimental */
final class CommandAwareInputDataProcessor implements CommandAwareInputDataProcessorInterface
{
    /** @param CommandDataProcessorInterface[] $commandDataProcessors */
    public function __construct(
        private iterable $commandDataProcessors,
    ) {
    }
```
I'm not sure, but have you tried in this way? Instead of dynamic property - just mark it as a class field